pkgs/development/libraries/libhttpseverywhere/default.nix

```diff
@@ -1,13 +1,12 @@
-{stdenv, fetchFromGitHub, gnome3, glib, json_glib, libxml2, libarchive, libsoup, gobjectIntrospection, meson, ninja, pkgconfig,  valadoc}:
+{stdenv, fetchgit, gnome3, glib, json_glib, libxml2, libarchive, libsoup, gobjectIntrospection, meson, ninja, pkgconfig,  valadoc}:
 
 stdenv.mkDerivation rec {
   name = "libhttpseverywhere-${version}";
   version = "0.2.3";
 
-  src = fetchFromGitHub {
-    owner = "grindhold";
-    repo  = "libhttpseverywhere";
-    rev = "${version}";
+  src = fetchgit {
+    url    = "https://git.gnome.org/browse/libhttpseverywhere";
```

Comment:
There is also an archive, which can be downloaded instead: https://git.gnome.org/browse/libhttpseverywhere/refs/